src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/PetCreator.java

```diff
@@ -74,12 +86,14 @@ public PetTsvCreator(String sampleIdentifierForOutputFileName, String sampleId,
             }
 
             if (writeReferenceRanges) {
-                final File refOutputFile = new File(outputDirectory, REF_RANGES_FILETYPE_PREFIX + tableNumberPrefix + sampleIdentifierForOutputFileName + ".avro");
+                final File refOutputFile = new File(outputDirectory, REF_RANGES_FILETYPE_PREFIX + tableNumber + PREFIX_SEPARATOR + sampleIdentifierForOutputFileName + ".avro");
                 refRangesAvroWriter = new RefRangesAvroWriter(refOutputFile.getCanonicalPath());
             }
 
         } catch (final IOException e) {
             throw new UserException("Could not create pet outputs", e);
+        } catch (final Exception ex) {
```

Comment:
can we be more specific in the exception types here?  and use a multi-exception catch if we handle them the same way (including IOException above?)